### DISCUSSION TITLE
How do I make my graphs match the OpenMage screenshots?

### DISCUSSION THREAD

**Discussion by durzel** [OpenMage/magento-lts#3105] [Questions & Answers] [answered]
Hi,

Hopefully a simple one this.  I'm running OpenMage 20.0.20, having upgraded a M1 1.9.4.5 install.  I vaguely remember having to fix the dashboard graphs a long time ago that stopped working spontaneously, but I'm not 100% on that.

Anyway, looking at the OpenMage homepage one of the screenshots looks like this:

![Screenshot 2023-03-20 at 12 22 27](https://user-images.githubusercontent.com/7250950/226338239-f57603bb-238b-48d3-9d3c-3c7a923852bb.png)

My dashboard graph doesn't look like that at all, it just looks like the old school orange image-charts.com one.  There's no configuration that I can see in the backend to change the service used for graphs, only "Enable Charts" as yes or no.

Am I missing something obvious here?

Thanks in advance :)

**Comment by sreichel** [comment, marked as answer]
There are no graphs like this. Sadly.

**Reply by durzel** [reply]
Really?  That sucks.  Seems strange that OpenMage advertise it so prominently if it's just a mockup :(

**Reply by fballiano** [reply]
there's no "openmage" entity, it's us the people, and we need more people to join developing the project we like

**Reply by durzel** [reply]
Ok, to be clear - it is strange that there appears to be a screenshot on the homepage of the project showing a dashboard with a graph that isn't the default and apparently doesn't exist.  I don't think it's unreasonable to expect "OpenMage Admin panel in action" screenshots to be actual screenshots instead of mockups?

**Reply by fballiano** [reply]
probably it was a test that didn't get finalized or something, the website is on github anyway, anybody can open a PR a fix a problem there too.

**Reply by durzel** [reply]
Thanks.  I am well aware of the mechanics of Github and how people can contribute.  I was just simply asking how I reproduce the graphs shown on www.openmage.org, and it has now been made clear to me that this isn't a configuration error on my behalf, or file conflicts or whatever, but just "false advertising", so to speak.

It's not a hill I'm going to die on, I have my answer now - thanks :)

**Reply by fballiano** [reply]
I just said it because not everybody knows that the website is also a github project...

please open an issue there and maybe somebody will fix it, we NEED people to contribute, if everybody doesn't care there's no openmage later.

**Reply by elidrissidev** [reply]
Might be easily achievable using a library like [D3](https://d3js.org/).

**Reply by elidrissidev** [reply]
Feature Request created: #3121
